python/ray/rllib/models/catalog.py

```diff
@@ -34,6 +35,9 @@ class ModelCatalog(object):
         action_op = dist.sample()
     """
 
+    ATARI_OBS_SHAPE = (210, 160, 3)
```

Comment:
A little odd that these dimensions are in the Model Catalog, but it's fine for now.